certbot/certbot/_internal/cert_manager.py

```diff
@@ -90,9 +90,16 @@ def certificates(config):
 def delete(config):
     """Delete Certbot files associated with a certificate lineage."""
     certnames = get_certnames(config, "delete", allow_multiple=True)
+    disp = zope.component.getUtility(interfaces.IDisplay)
+    msg = ["The following certificate(s) is/are selected for deletion:\n"]
```

Comment:
Sorry for the delay! Let's get this merged.

I think it's OK to just use the plural form here.

```suggestion
    msg = ["The following certificate(s) are selected for deletion:\n"]
```